react/features/visitors/actions.ts

```diff
@@ -19,13 +19,11 @@ export function admitMultiple(requests: Array<IPromotionRequest>): Function {
     return (dispatch: IStore['dispatch'], getState: IStore['getState']) => {
         const conference = getCurrentConference(getState);
 
-        requests.forEach(r => {
-            conference?.sendMessage({
-                type: 'visitors',
-                action: 'promotion-response',
-                approved: true,
-                id: r.from
-            });
+        conference?.sendMessage({
+            type: 'visitors',
+            action: 'promotion-response',
+            approved: true,
+            ids: requests.map(r => r.from)
```

Comment:
I'm wondering wouldn't it will be better if we always have only `ids` and for the use case of a single request to pass only 1 id in the array?

Reply:
This will break mobile that needs to be updated. Now backend handles both.